index.html

```diff
@@ -2570,6 +2605,16 @@ <h3>
 
       </section>
 
+       <section>
+         <h3>
+ Reserved Properties
+         </h3>
+
+         <p>
+This representation format reserves the <code>@context</code> property.
```

Comment:
If we are going to add this section, which is a terrible idea, here's the full list of reserved properties:

https://www.w3.org/TR/json-ld/#syntax-tokens-and-keywords

So that means: 
```
@base 
@container 
@context 
@direction 
@graph 
@id 
@import 
@included 
@index 
@json 
@language 
@list 
@nest 
@none 
@prefix 
@propagate 
@protected 
@reverse 
@set 
@type 
@value 
@version 
@vocab 
```

This is going to complicate JSON-only processors to the point where I would question whether implementing a JSON-only processor would be worth it.

The simpler approach is just to preserve everything when going from one format to the abstract data model, and if you can't, throw an error.

Reply:
I should also point out that this PR will require us to have prior knowledge of other representation formats... for example, if someone were to come up with a JSON Schema representation format (in order to include a signed validation mechanism with their DID Documents), then we'd also have to make sure that the JSON-only implementation would need to be aware of the following:

```
string
number
integer
boolean
object
array
null
id
type
description
default
patternProperties
dependencies
extends
$ref
$schema
allOf, anyOf, oneOf
enum
format
minLength, maxLength
pattern
minimum, maximum
exclusiveMinimum, exclusiveMaximum
multipleOf
array
items
minItems, maxItems
uniqueItems
additionalItems
object
properties
minProperties, maxProperties
patternProperties
additionalProperties
required
```

... we're clearly not going to do that as the implementation burden is very high *and* it conflicts directly with some of the property names we've already picked for the DID Core abstract data model (such as `id`, `type`, etc.). The reason that it would be fine to preserve these things is because nesting matters. JSON, JSON-LD, and other JSON formats would be just fine w/ embedded JSON Schema (and embedded JSON-LD) because those formats were designed around two primary features of JSON -- not dropping anything and nested objects providing context.

Reply:
I don't buy the slippery slope argument here. It's not true that if we do one thing we must do everything possible forever in all time.

Reply:
well sure, we can have a `deny-list` that is short, or a long one... both are still worse than having an `allow-list`... IMO, probably best to leave the security engineering to the did method authors... I don't have confidence this WG has the expertise necessary to design a safe `deny-list`.